internal/functions/efgs/database.go

```diff
@@ -0,0 +1,150 @@
+package efgs
+
+import (
+	"context"
+	"errors"
+	"github.com/covid19cz/erouska-backend/internal/logging"
+	"net"
+
+	"github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/proxy"
+	"github.com/covid19cz/erouska-backend/internal/secrets"
+	"github.com/go-pg/pg/v10"
+	"github.com/go-pg/pg/v10/orm"
+)
+
+//Singleton database struct. Must be initialize with CreateDatabaseConnection!
+var database Database
+
+//Database Contains database connection pool.
+type Database struct {
+	connection *pg.DB
+}
+
+//CreateDatabaseConnection Create new database connection pool. Credentials must be specified in evn variables.
+func CreateDatabaseConnection() (Database, error) {
```

Comment:
I believe this should be done in the `init()` function, like in [Firebase clients](https://github.com/covid19cz/erouska-backend/blob/main/internal/firebase/firebase.go#L23) - see [tutorial](https://tutorialedge.net/golang/the-go-init-function/).

Reply:
Ok, sounds reasonable.